ocs_ci/ocs/resources/ocs.py

```diff
@@ -97,11 +97,12 @@ def create(self, do_reload=True):
         return status
 
     def delete(self, wait=True, force=False):
-        result = self.ocp.delete(
-            resource_name=self.name, wait=wait, force=force
-        )
-        self._is_deleted = True
-        return result
+        if not self.is_deleted:
```

Comment:
Should we log when we aren't deleting due to `self._is_deleted` already being `True`? Even if it's `debug` level it's probably worth noting.

Reply:
Sure, I think info level is more helpful too, since its not that many lines